crates/cxx-qt-lib/src/qml/qqmlapplicationengine.rs

```diff
@@ -73,6 +73,11 @@ mod ffi {
             qobject: *mut QObject,
             url: &QUrl,
         );
+
+        #[qsignal]
+        #[rust_name = "object_creation_failed"]
+        #[cfg(any(cxxqt_qt_version_at_least_7, cxxqt_qt_version_at_least_6_4))]
+        unsafe fn objectCreationFailed(self: Pin<&mut QQmlApplicationEngine>, url: &QUrl);
```

Comment:
this doesn't need to be `unsafe` as it doesn't have a pointer in it's argument. Otherwise looks good.